chromium_src/chrome/browser/ui/cocoa/first_run_dialog_controller.mm

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/grit/brave_generated_resources.h"
+#include "chrome/grit/generated_resources.h"
+
+// Replaced string here instead of by running 'npm run chromium_rebase_l10n'
+// because string replacement failed with string that include place holder like
+// <ph name="PRODUCT_NAME">$1<ex>Brave</ex></ph>. I assume this include some
+// special charactors.
```

Comment:
would be good to spellcheck the comment: `include`->`includes`, `charactors` -> `characters`